sentry/src/main/java/io/sentry/metrics/MetricType.java

```diff
@@ -1,12 +1,19 @@
 package io.sentry.metrics;
 
 import org.jetbrains.annotations.ApiStatus;
+import org.jetbrains.annotations.NotNull;
 
 /** The metric instrument type */
 @ApiStatus.Internal
 public enum MetricType {
-  Counter,
-  Gauge,
-  Distribution,
-  Set
+  Counter("c"),
+  Gauge("g"),
+  Distribution("d"),
+  Set("s");
+
+  final @NotNull String statsdCode;
+
+  MetricType(final @NotNull String statsdCode) {
+    this.statsdCode = statsdCode;
```

Comment:
`nit`: I separated `MetricType` from Statsd on purpose: If we ever decide to use a different protocol, we would only need to refactor some helpers and transport specific code. But I'm still fine with doing it this way, as that's what other SDKs are doing as well.